dds_web/api/db_connector.py

```diff
@@ -84,9 +96,15 @@ def project_size(self):
 
         num_proj_files, error = (0, "")
         try:
+            current_project = models.Project.query.filter(
+                models.Project.public_id == func.binary(self.project["id"])
+            ).first()
```

Comment:
Yes, but I'm looking into a better way - not getting the current project first, just using `join`. So this was just temporary, so that we have the db structure and then I can fix a better way.